sherpa-onnx/python/csrc/online-model-config.cc

```diff
@@ -27,14 +27,16 @@ void PybindOnlineModelConfig(py::module *m) {
       .def(py::init<const OnlineTransducerModelConfig &,
                     const OnlineParaformerModelConfig &,
                     const OnlineWenetCtcModelConfig &,
-                    const OnlineZipformer2CtcModelConfig &, const std::string &,
-                    int32_t, bool, const std::string &, const std::string &>(),
+                    const OnlineZipformer2CtcModelConfig &,
+                    const std::string &, int32_t, int32_t,
+                    bool, const std::string &, const std::string &>(),
            py::arg("transducer") = OnlineTransducerModelConfig(),
            py::arg("paraformer") = OnlineParaformerModelConfig(),
            py::arg("wenet_ctc") = OnlineWenetCtcModelConfig(),
            py::arg("zipformer2_ctc") = OnlineZipformer2CtcModelConfig(),
-           py::arg("tokens"), py::arg("num_threads"), py::arg("debug") = false,
-           py::arg("provider") = "cpu", py::arg("model_type") = "")
+           py::arg("tokens"), py::arg("warm_up") = 0, py::arg("num_threads"),
```

Comment:
please follow the order at
https://github.com/k2-fsa/sherpa-onnx/blob/bdb74815d8a0b9dd427a0614453050b78b3243f6/sherpa-onnx/csrc/online-model-config.h#L21-L24

It should be
```cpp
py::arg("tokens"),
py::arg("num_threads") = 1,
py::arg("warm_up") = 0,
py::arg("debug") = false,
```

Reply:
Taken care.